package.json

```diff
@@ -14,15 +15,23 @@
   },
   "author": "Grant Timmerman",
   "license": "MIT",
+  "engines": {
+    "node": ">=8.10.0"
+  },
   "bugs": {
     "url": "https://github.com/grant/ts2gas/issues"
   },
   "homepage": "https://github.com/grant/ts2gas#readme",
   "dependencies": {
+    "lodash": "^4.17.11",
     "typescript": "^3.2.2"
   },
   "devDependencies": {
     "tslint": "^5.11.0",
+    "@types/lodash": "^4.14.119",
     "@types/node": "^10.12.18"
+  },
+  "peerDependencies": {
+    "@google/clasp": "^1.7.0"
```

Comment:
We don't need this peer dep.

Reply:
TBH, I wasn't sure if it would help at all :)